blocks/small-promo-block/features/small-promo/default.jsx

```diff
@@ -1,16 +1,13 @@
 import React from 'react';
 import PropTypes from '@arc-fusion/prop-types';
-import { useContent, useEditableContent } from 'fusion:content';
+import { useContent } from 'fusion:content';
 import { useFusionContext } from 'fusion:context';
 import { LazyLoad, isServerSide } from '@wpmedia/engine-theme-sdk';
 import { imageRatioCustomField } from '@wpmedia/resizer-image-block';
-import {
-  PromoHeadline, PromoImage, SmallPromoContainer, SmallPromoStyles,
-} from '@wpmedia/shared-styles';
+import { SmallPromoPresentation } from '@wpmedia/shared-styles';
 
 const SmallPromoItem = ({ customFields }) => {
-  const { arcSite, isAdmin } = useFusionContext();
-  const { searchableField } = useEditableContent();
+  const { arcSite } = useFusionContext();
```

Comment:
I'm indifferent to this, but Matt brought this up on one of my reviews... we can just pass arcSite in from the parent to avoid another lookup of useFusionContext here.

Reply:
This is the parent component in this instance

Reply:
My bad - I just looked at the file name - this should be passed in on line 88